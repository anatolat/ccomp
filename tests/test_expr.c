
int main() {
	int a = 10;
	int b = 5;
	
	// arithmetic
	printf("a+b=%d a-b=%d a*b=%d a/b=%d a%%b=%d\n",
		a + b,
		a - b,	
		a * b,
		a / b,
		a % b);	

	// unary not
	printf("!0=%d !1=%d\n", !0, !1);
}