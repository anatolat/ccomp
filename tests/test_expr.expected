a+b=15 a-b=5 a*b=50 a/b=2 a%b=0
!0=1 !1=0
